### PR TITLE
[ICDS Dashboard] Adding missing fields for child health vaccines in cas export

### PR DESCRIPTION
**Pull request by challabeehyv**
https://app.asana.com/0/0/1149928009344929/f

##### SUMMARY
updated fields in child tasks json and updated the fixtures accordingly

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/tests/agg_tests/fixtures/child_tasks.csv [hunk 1]
```diff
@@ -1,134 +1,134 @@
...
+task_id_02,2017-03-01,st1,a44,0a04f052-f32b-4874-b38b-d21244f81516,child,,,2017-03-01,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,s4,2017-03-01,2017-03-01,2017-03-01,2017-03-01,2017-03-01,2017-03-01,2017-03-01,2017-03-01
```
Comment (Rohit25negi) [inline custom/icds_reports/tests/agg_tests/fixtures/child_tasks.csv]: I dont see any of the test output data is updated of child_health table and mbt. Could you update the fixture such that this change can be verified.

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
left a few comments

File: custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql [hunk 2]
```diff
@@ -65,7 +65,51 @@ CREATE VIEW child_health_monthly_view AS
         WHEN (child_health_monthly.zscore_grading_wfh = 2 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 2 AND child_health_monthly.muac_grading_recorded_in_month = 2) THEN 2
         WHEN (child_health_monthly.zscore_grading_wfh = 3 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 3 AND child_health_monthly.muac_grading_recorded_in_month = 3) THEN 3
         ELSE 4 END AS current_month_wasting_v2_sort,
-      child_health_monthly.mother_phone_number
+      child_health_monthly.mother_phone_number,
```
[the remark this answers is not in the document]
Comment (challabeehyv) [reply custom/icds_reports/migrations/sql_templates/database_views/child_health_monthly.sql]: yes. It is taking from the table itself. I thought they need to be consistent. I will change this

File: custom/icds_reports/tests/agg_tests/fixtures/child_tasks.csv (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/tests/agg_tests/fixtures/child_tasks.csv]: i updated child_health_monthly_2017-04-01_sorted.csv and child_health_monthly_2017-05-01_sorted.csv. Do i have to add more changes?

**Comment by challabeehyv** [comment]
closing this as it is handled in the bihar vaccine pr https://github.com/dimagi/commcare-hq/pull/27120

**Pull request closed without merging**